### ISSUE TITLE
Survey relations used in GO-CAMs and their corresponding display in the pathway view

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/wc-gocam-viz#50]
@dustine32 @tmushayahama 

Can you link the relations spreadsheet Tremayne showed at last week's Noctua workbenches meeting to this ticket so we can keep track centrally of what the overall status is for this work?

Thanks!

**Comment by dustine32** [comment]
@vanaukenk I copied the relations from my query results to a new google sheet here: https://docs.google.com/spreadsheets/d/1Gt1rgPocGQ4uNKgzBhbp3vCJa-HRzzjPx0GQkwcj81Q/edit?usp=drive_link

@tmushayahama's spreadsheet may have more info.

**Comment by vanaukenk** [comment]
Thank you @dustine32 !

**Comment by vanaukenk** [comment]
@dustine32 - should we also add the CHEBI -> MF relations, i.e.:

'is small molecule inhibitor of'
'is small molecule activator of'

**Comment by vanaukenk** [comment]
See #33

**Comment by dustine32** [comment]
@vanaukenk Yep, my initial query didn't look for relations in the CHEBI -> MF direction. You can add those two to the spreadsheet and I'll try editing my query to get both directions.

**Comment by vanaukenk** [comment]
@dustine32 - the two small molecule regulator relations are now at the bottom of your spreadsheet.  Thx.

**Comment by dustine32** [comment]
@vanaukenk To confirm the final list of used CHEBI -> MF relations, I ran the updated query and those small molecule regulator relations are indeed the only relations found other than a few uses of "directly positively regulates" used in 7 models (e.g., [SP_GADD34- PPP1CA](http://noctua.geneontology.org/workbench/noctua-visual-pathway-editor/?model_id=gomodel:61e0e55600000407&barista_token=)).
